notebooks/ImvizExample.ipynb

```diff
@@ -92,69 +98,62 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "acs_47tuc_1 = download_file('https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jbqf03gjq_flc.fits', cache=True)\n",
```

Comment:
Can we keep the ACS example but in "concepts"? I rely on them a lot because it tests dithering with distortion. This use case has helped us catch quite a few bugs.

Reply:
The dithering consideration is a good point. I think I'll actually keep the alternate ACS notebook in the top level notebooks folder and point to it as an alternative if users don't want to download the large JWST files.